utils/supabase/DatabaseTypes.d.ts

```
@@ -317,6 +332,168 @@ export type HelpRequest = GetResult<
+
+export type HelpRequestLocationType = Database["public"]["Enums"]["location_type"];
+
+export type HelpRequestTemplate = GetResult<
+  Database["public"],
+  Database["public"]["Tables"]["help_request_templates"]["Row"],
+  "help_request_templates",
+  Database["public"]["Tables"]["help_request_templates"]["Relationships"],
+  "*"
+>;
+
+export type HelpRequestModeration = GetResult<
```

Biome (2.1.2)

[error] Shouldn't redeclare 'HelpRequestModeration'. Consider to delete it or rename it.

'HelpRequestModeration' is defined here:


(lint/suspicious/noRedeclare)